mldsa/polyvec.h

```diff
@@ -38,7 +55,17 @@ void polyvecl_reduce(polyvecl *v);
  *              - const polyvecl *u: pointer to first summand
  *              - const polyvecl *v: pointer to second summand
  **************************************************/
-void polyvecl_add(polyvecl *w, const polyvecl *u, const polyvecl *v);
+void polyvecl_add(polyvecl *w, const polyvecl *u, const polyvecl *v)
+__contract__(
+  requires(memory_no_alias(w, sizeof(polyvecl)))
```

Comment:
This is not going to work, and similar for `poly_add`, since the function is being called with aliasing first and second argument -- like in mlkem-native.